index.html

```diff
@@ -0,0 +1,350 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="/css/style.css">
+  </head>
+  <body>
+    <nav class="menu">
+      <a href="#home">Home.</a>
+      <a href="#services">Services.</a>
+      <a href="#portfolio">Portfolio.</a>
+      <a href="#about">About.</a>
+      <a href="#team">Team.</a>
+      <a href="#contact">Contact.</a>
```

Comment:
Although this is valid, it's good practice to use an unordered list (`ul`) to manage a list of links. While it means more CSS management to product a layout like this, when you look at the raw document (no styles applied), the links have a more logical layout _and_ it gives you greater flexibility if/when you need to change the layout to a list-view (which is commonly done when using an off-screen menu for mobile views).